### ISSUE TITLE
Allow approval of deposits via the review workflow that are now past the embargo date set by the depositor.

### ISSUE THREAD

**Issue by amyehodge** [sul-dlss/happy-heron#952]
The scenario here is that a student will sometimes set an embargo for a very short time in the future, like one week. By the time the deposit makes it through the review workflow and the reviewer is ready to approve it, the embargo date is now in the past. This normally causes an error that would say the embargo date can't be in the past.

Ideally what would happen in this situation in the review workflow is that when the reviewer clicks the deposit button if the embargo date is now in the past, it will just be ignored and the item will be deposited and released as if the release were set for "immediately." No information about the embargo dates set by the depositor need to be stored in the system.

Please let me know if there any questions about this.

**Comment by peetucket** [comment]
Note: approach can be flexible to make it more efficient if complicated validation is not advisable or too time consuming -- check in with Amy when picked up to provide options if necessary.

**Comment by jcoyne** [comment]
@amyehodge in my testing, I was able to approve a deposit that had the embargo date in the past without issue.  Do we need to do anything further?

**Comment by amyehodge** [comment]
@jcoyne See #543. I think you can approve this because embargo dates are currently being allowed in the past but shouldn't be. Once a fix for that is implemented, we need a way around it for this situation. Hopefully that makes sense.

**Comment by jcoyne** [comment]
@amyehodge I don't think that's true. Editing a deposit is a separate mechanism from approving a deposit. The approval doesn't do any validation as this was already done by the depositor.

**Comment by amyehodge** [comment]
@jcoyne This came up as a known issue in Hydrus originally. Perhaps we should wait on this until #543 is completed and then I can check back on this to see if it's still working as desired.

**Comment by jcoyne** [comment]
@amyehodge I think that's a great idea. I'll put this in the backlog and assign to you.

**Comment by amyehodge** [comment]
This is working as desired now. Closing ticket.
